{
	"affected_release": [
		{
			"advisory": "RHSA-2021:0778",
			"cpe": "cpe:/a:redhat:ansible_tower:3.6::el7",
			"impact": "moderate",
			"package": "ansible-tower-36/ansible-tower:3.6.7-1",
			"product_name": "Red Hat Ansible Tower 3.6 for RHEL 7",
			"release_date": "2021-03-09T00:00:00Z"
		},
		{
			"advisory": "RHSA-2021:0779",
			"cpe": "cpe:/a:redhat:ansible_tower:3.7::el7",
			"impact": "moderate",
			"package": "ansible-tower-37/ansible-tower-rhel7:3.7.5-1",
			"product_name": "Red Hat Ansible Tower 3.7 for RHEL 7",
			"release_date": "2021-03-09T00:00:00Z"
		},
		{
			"advisory": "RHSA-2021:0780",
			"cpe": "cpe:/a:redhat:ansible_automation_platform:3.8::el7",
			"impact": "moderate",
			"package": "ansible-tower-38/ansible-runner-rhel7:1.4.7-1",
			"product_name": "Red Hat Ansible Tower 3.8 for RHEL 7",
			"release_date": "2021-03-09T00:00:00Z"
		},
		{
			"advisory": "RHSA-2021:0780",
			"cpe": "cpe:/a:redhat:ansible_automation_platform:3.8::el7",
			"impact": "moderate",
			"package": "ansible-tower-38/ansible-tower-rhel7:3.8.2-1",
			"product_name": "Red Hat Ansible Tower 3.8 for RHEL 7",
			"release_date": "2021-03-09T00:00:00Z"
		}
	],
	"bugzilla": {
		"description": "CVE-2020-35678 python-autobahn: allows redirect header injection",
		"id": "1911314",
		"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1911314"
	},
	"csaw": false,
	"cvss3": {
		"cvss3_base_score": "6.1",
		"cvss3_scoring_vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:C/C:L/I:L/A:N",
		"status": "verified"
	},
	"cwe": "CWE-601",
	"details": [
		"Autobahn|Python before 20.12.3 allows redirect header injection.",
		"A flaw was found in python-autobahn, where it allows redirect header injection. The highest threat from this vulnerability is to confidentiality and integrity."
	],
	"mitigation": {
		"lang": "en:us",
		"value": "Mitigation for this issue is either not available or the currently available options do not meet the Red Hat Product Security criteria comprising ease of use and deployment, applicability to widespread installation base or stability."
	},
	"name": "CVE-2020-35678",
	"package_state": [
		{
			"cpe": "cpe:/a:redhat:ansible_automation_platform",
			"fix_state": "Affected",
			"impact": "moderate",
			"package_name": "autobahn",
			"product_name": "Red Hat Ansible Automation Platform 1.2"
		},
		{
			"cpe": "cpe:/a:redhat:openstack:10",
			"fix_state": "Out of support scope",
			"package_name": "python-autobahn",
			"product_name": "Red Hat OpenStack Platform 10 (Newton)"
		},
		{
			"cpe": "cpe:/a:redhat:openstack:13",
			"fix_state": "Will not fix",
			"package_name": "python-autobahn",
			"product_name": "Red Hat OpenStack Platform 13 (Queens)"
		},
		{
			"cpe": "cpe:/a:redhat:openstack:16.1",
			"fix_state": "Will not fix",
			"package_name": "python-autobahn",
			"product_name": "Red Hat OpenStack Platform 16.1"
		},
		{
			"cpe": "cpe:/a:redhat:quay:3",
			"fix_state": "Will not fix",
			"package_name": "quay",
			"product_name": "Red Hat Quay 3"
		}
	],
	"public_date": "2020-12-18T00:00:00Z",
	"references": [
		"https://www.cve.org/CVERecord?id=CVE-2020-35678\nhttps://nvd.nist.gov/vuln/detail/CVE-2020-35678"
	],
	"statement": "In Red Hat OpenStack Platform, because the flaw has a lower impact and the fix would require a substantial amount of development, no update will be provided at this time for the RHOSP python-autobahn package.",
	"threat_severity": "Moderate",
	"upstream_fix": "python-autobahn 20.12.3"
}
